libr/bp/bp.c

```diff
@@ -1,6 +1,7 @@
 /* radare2 - LGPL - Copyright 2009-2018 - pancake */
 
 #include <r_bp.h>
+#include <r_debug.h>
```

Comment:
Nope

Reply:
So should I add isValidMap in corebind that checks address + permissions?